doc/pundit-notes.org

```diff
@@ -0,0 +1,190 @@
+#+TITLE: pundit-notes
```

Comment:
Not entirely sure. It's still a useful reference for now, because I don't do any cleaning up of links in this PR, that's why I kept it in for the time being.

Reply:
up to you